### PR TITLE
added routes

### PR DESCRIPTION
**Pull request by danielnaumau**

### CODE REVIEW

**Comment by danielnaumau** [comment]
Related https://github.com/traveltime-dev/traveltime-benchmarks/pull/48

**Comment by ajakubauskas** [comment]
missing entry on routes in README line 126

**Comment by danielnaumau** [comment]
> README

image

what entry?

**Comment by ajakubauskas** [comment]
> > README
> 
> image
> what entry?
Screenshot from 2024-03-08 12-08-55

```
### Transport modes
#### time-map, time-map-fast, time-filter
```
I'm assuming it should also have `routes` and the following modes are applicable, right?

File: scripts/routes.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    transportation,
+    countryCoords,
+    dateTime
+) {
+    const coordinates = countryCoords
+    const diff = 0.01
+
+    const origin = {
+        id: 'origin',
+        coords: generateRandomCoordinate(coordinates.lat, coordinates.lng, diff)
```
Comment (ajakubauskas) [inline scripts/routes.js:85]: something's messed up here, got runtime errors:
```
ERRO[0006] TypeError: Cannot read property 'lat' of undefined
running at generateBody (file:///home/aivaras/Work/traveltime-benchmarks/scripts/routes.js:85:53(12))
mainScenat file:///home/aivaras/Work/traveltime-benchmarks/scripts/routes.js:51:49(60)  executor=constant-arrival-rate scenario=mainScenario source=stacktrace
```

Comment (ajakubauskas) [reply scripts/routes.js:85]: in fact other endpoints seem to have this issue as well:
```
ERRO[0006] TypeError: Cannot read property 'lat' of undefined
running at generateBody (file:///home/aivaras/Work/traveltime-benchmarks/scripts/time-map.js:76:51(14))
mainScenat file:///home/aivaras/Work/traveltime-benchmarks/scripts/time-map.js:49:47(68)  executor=constant-arrival-rate scenario=mainScenario source=stacktrace
```

Comment (danielnaumau) [reply scripts/routes.js:85]: how are you trying to run it?

Comment (ajakubauskas) [reply scripts/routes.js:85]: > how are you trying to run it?

`k6 run -e APP_ID=xxx -e API_KEY=yyy -e COUNTRY=us_estn -e RPM=10 scripts/routes.js `

Comment (danielnaumau) [reply scripts/routes.js:85]: wrong country code

Comment (danielnaumau) [reply scripts/routes.js:85]: for json requests it's just us

**Review by ajakubauskas — APPROVED** [review]

**Pull request merged**